src/olympia/activity/models.py

```diff
@@ -41,6 +41,18 @@
 
 GENERIC_USER_NAME = gettext('Add-ons Review Team')
 
+# Activity ids that are not considered as needing a reply from developers, so
+# they are never considered "pending".
+NOT_PENDING_IDS = (
```

Comment:
I see it predates this PR but curious why a negative definition of NOT_PENDING instead of PENDING_IDS?

Reply:
@eviljeff might remember but I don't know, I haven't touched that code before. I think I would have preferred a property on the activity constant like we do elsewhere but I didn't want to expand the scope too much.

Reply:
I don't remember either.  I think if I was writing this today I'd start with the activities that could need a reply instead (from properties in the constant).